### DISCUSSION TITLE
v0.1.0 upgrading steps

### DISCUSSION THREAD

**Discussion by noraj** [pwndoc/pwndoc#127] [Q&A]
There some breaking changes https://github.com/pwndoc/pwndoc/releases/tag/v0.1.0

Any steps to follow for upgrading?

**Comment by yeln4ts** [comment]
No particular steps, just take into account what is indicated in the Breaking Change section (Save your custom sections default text and create any section you need in your pending audits)
I advise to always make a backup of your old pwndoc folder before updating so that you can always come back to a known working version if there are any problems
